sycl/doc/extensions/supported/README.md

```diff
@@ -0,0 +1,10 @@
+This directory contains the specifications for DPC++ extensions that are fully
+supported.  The APIs in these extensions are generally stable in future
```

Comment:
```suggestion
This directory contains the specifications for SYCL extensions that are fully
supported by DPC++ implementation.  The APIs in these extensions are generally stable in future
```

Reply:
Makes sense.  Changed in a45767a0f34631b05767f68354ebbab3bbc19c63.